### PR TITLE
RESTEASY-2737 - creating an example showing how to deploy resteasy-wa…

### PR DESCRIPTION
**Pull request by liweinan**
…dl to wildfly

### CODE REVIEW

**Comment by liweinan** [comment]
This depends on:

- [RESTEASY-2734 The current WADL GRAMMAR generation logic incorrectly c… by liweinan · Pull Request #2568 · resteasy/Resteasy · GitHub](https://github.com/resteasy/Resteasy/pull/2568)


@ronsigal @asoldano  Please help to check.

File: resteasy-wadl-wildfly/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+## USAGE
+
+To deploy this sample project into Wildfly, firstly you need to upgrade your Wildfly RESTEasy module to use the same 
+version with the RESTEasy version used in this example. To achieve the goal, please refer to this section of the RESTEasy
+document:
+
+* [Upgrading RESTEasy within WildFly](https://docs.jboss.org/resteasy/docs/4.5.8.Final/userguide/html_single/index.html#upgrading-wildfly)
+
+After upgrading the Wildfly RESTEasy module to same version of the RESTEasy dependency of this project, the next step is
+to package this project with the following Maven command:
```
Comment (asoldano) [inline resteasy-wadl-wildfly/README.md:12]: I'm wondering if running on at least the same version of RESTEasy pulled by the examples is not always a pre-requisite and hence this would have to be removed from here. WDYT?

File: pom.xml [hunk 2]
```diff
@@ -61,7 +62,8 @@
         <maven.compiler.target>8</maven.compiler.target>
         <persistence-api.version>1.0.2</persistence-api.version>
         <resteasy-extension.ver>1.0.0.Final</resteasy-extension.ver>
-        <resteasy.ver>4.5.6.Final</resteasy.ver>
+<!--        <resteasy.ver>4.5.6.Final</resteasy.ver>-->
```
Comment (asoldano) [inline pom.xml:65]: Remove commented out line maybe

**Review by asoldano — COMMENTED** [review]
Thanks @liweinan , please see my comments

**Comment by liweinan** [comment]
@asoldano thanks for reviewing! I'll wait for newer version of RESTEasy to be released, and then update the dep resteasy version in this PR, and then do the resteasy-examples release accordingly

File: pom.xml (hunk 2 above)
Comment (liweinan) [reply pom.xml:65]: I'll change it to the latest release we have right now.

File: resteasy-wadl-wildfly/README.md (hunk 1 above)
Comment (liweinan) [reply resteasy-wadl-wildfly/README.md:12]: Let me rephrase the test here.

**Comment by liweinan** [comment]
@asoldano I have modified the places you've referred to. Could you please to help to check it again? Thanks!

After it's approved I'll merge this and write blog + update resteasy doc according to the example.

**Review by asoldano — APPROVED** [review]
